ci/docker/serve.build.Dockerfile

```diff
@@ -16,6 +16,14 @@ SHELL ["/bin/bash", "-ice"]
 
 COPY . .
 
+# Install podman
+RUN sudo apt update
```

Comment:
1. please add the deb source first, and then update only ones, and then install all packages they you need in one line.
2. move these before the `COPY . .`

and we are using buildkit, so consider using a multiline. see the other exising `RUN <<EOF` for example. multiline will generate fewer image layers and is faster to load.

Reply:
Let's fix this